prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -43,9 +43,15 @@ elements:
       csvFields: [{ name: Patient Ethnicity }]
 
     # column I
-    # TODO: does this need a custom mapper? ELR FAQ v2.6 has CSV specific values for HL7 codes
     - name: patient_gender
       csvFields: [{ name: Patient Gender }]
+      altValues:
```

Comment:
In cases where there are lots of different ways of saying the same thing, it occurs to me that we should move to using @RickHawesUSDS 's `TABLE` type rather than CODE.    Then each time a state has a new "creative" way of specifying something we can just add a column to the table.